COMPATIBILITY.md

```diff
@@ -3,25 +3,23 @@
 
 ## Compatibility with OpenSearch
 
-The below matrix shows the compatibility of the [`opensearch-ruby`](https://rubygems.org/gems/opensearch-ruby) with versions of [`OpenSearch`](https://opensearch.org/downloads.html#opensearch).
+The OpenSearch Ruby client is comprised of three gems:
+- [`opensearch-transport`](https://rubygems.org/gems/opensearch-transport) - the transport layer
```

Comment:
@nhtruong Do you think we also need to include the individual gem versions as well for compatibility? May be their compatibility with the main gem `opensearch-ruby`. For example, this file mentions the compatibility of `opensearch-ruby` with OpenSearch. And each individual gem folder has compatibility matrix showing compatibility with `opensearch-ruby` gem.

WDYT?

Reply:
You're right. Lemme do some more research on the git history and add them. It's a bit tricky but once we have these 3 gems merged into, it will be...history!